Button/example.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Back, Close, Hamburger, Options, Search } from '../IconButton'
+import { Back, Close, Select, Hamburger, Options, Search } from '../IconButton'
```

Comment:
This import should be also in the example import statement

<img width="953" alt="screen shot 2017-01-22 at 11 19 43" src="https://cloud.githubusercontent.com/assets/381614/22181415/b63c8fa2-e094-11e6-8833-9d9e2e6e78a4.png">